src/main/java/com/github/jnidzwetzki/bitfinex/v2/entity/currency/BitfinexCurrencyPair.java

```diff
@@ -255,7 +255,7 @@ public static BitfinexCurrencyPair fromSymbolString(final String symbolString) {
 	 */
 	@Override
 	public String toBitfinexString() {
-		if(currencyType == BitfinexCurrencyType.PERPETUAL) {
+		if (currency1.length() > 3 || currency2.length() > 3) {
```

Comment:
Could you add this logic to the parseCurrencyType() method and set `BitfinexCurrencyType.PERPETUAL` in this case?